xrmocap/data/dataset/fourd_dataset.py

```diff
@@ -0,0 +1,386 @@
+                    view_idxs.append(
+                        int(
+                            re.search(r'fisheye_param_([0-9]+).json',
+                                      file_name).group(1)))
+            n_view = len(cam_names)
+            self.view_idxs = view_idxs
+            mview_list = []
+            for idx in range(n_view):
+                view_idx = self.view_idxs[idx]
+                fisheye_param = FisheyeCameraParameter.fromfile(
+                    os.path.join(cam_dir,
+                                 f'fisheye_param_{view_idx:02d}.json'))
+                translation = self.factor * np.array(
+                    fisheye_param.get_extrinsic_t())
+                fisheye_param.set_KRT(T=translation)
+                if fisheye_param.world2cam != self.cam_world2cam:
+                    fisheye_param.inverse_extrinsic()
+                mview_list.append(fisheye_param)
+            mscene_list.append(mview_list)
+        self.fisheye_params = mscene_list
+
+    def load_ground_truth_3d(self):
+        """Load multi-scene ground truth keypoints3d."""
+        mscene_list = []
+        for scene_idx in range(self.n_scene):
+            keypoints3d_path = os.path.join(self.meta_path,
+                                            f'scene_{scene_idx}',
+                                            'keypoints3d_GT.npz')
+            keypoints3d = Keypoints.fromfile(keypoints3d_path)
+            keypoints3d.logger = self.logger
+            keypoints3d = keypoints3d.to_tensor()
+            if self.gt_kps3d_convention is not None:
+                keypoints3d = convert_keypoints(
+                    keypoints=keypoints3d,
+                    dst=self.gt_kps3d_convention,
+                    approximate=True)
+            # save mask info into confidence
+            kps3d_mask = keypoints3d.get_mask()
+            kps3d = keypoints3d.get_keypoints()
+            kps3d[..., -1] = kps3d[..., -1] * kps3d_mask
+            kps3d[..., :3] = self.factor * kps3d[..., :3]
+            keypoints3d.set_keypoints(kps3d)
+            mscene_list.append(keypoints3d)
+        self.gt3d = mscene_list
+
+    def load_perception_2d(self):
+        """Load multi-scene keypoints2d and paf."""
+
+        mscene_keypoints_list = []
+        for scene_idx in range(self.n_scene):
+
+            pkl_names = glob.glob(
+                os.path.join(self.meta_path, f'scene_{scene_idx}', "kps2d_paf", '*pkl'))
+            n_view = len(pkl_names)
+            mview_kps2d = []
+            for view_idx in range(n_view):
+
+                img_size = (self.fisheye_params[scene_idx][view_idx].width, self.fisheye_params[scene_idx][view_idx].height)
+
+                file_name = os.path.join(self.meta_path, f'scene_{scene_idx}', "kps2d_paf", str(view_idx)+'.pkl')
+                f = open(file_name,'rb')
+                detections = pickle.load(f)
+                
+                frame_size = len(detections)
+
+                joint_size = 25
```

Comment:
Can we save these constant in the data file instead of defining them in code? We can load them like `joint_size = file_dict['joint_size']`. Otherwise, the data file is not completed if there's no code.